jekyll/_cci2/local-cli-getting-started.md

```diff
@@ -152,7 +152,7 @@ Success!
 	
 ## Connect Your Repo to CircleCI
 
-We will need to leave the terminal behind for this step. Head over to [the "Add Projects page"](https://circleci.com/add-projects). It's time to set up your project to run CI whenever you push code.
+We will need to leave the terminal behind for this step. Head over to [the "Add Projects page"](https://circleci.com/docs/2.0/project-build/#adding-projects). It's time to set up your project to run CI whenever you push code.
```

Comment:
I think it would be better to link to the new project dashboard here: https://app.circleci.com/projects/project-dashboard/github/circleci/